### ISSUE TITLE
[PROPOSAL] Command line option for source / dest folders

### ISSUE THREAD

**Issue by underparnv** [tightenco/jigsaw#24]
Would be cool to have an option to specify the [source](https://github.com/tightenco/jigsaw/blob/master/jigsaw#L29) and [destination](https://github.com/tightenco/jigsaw/blob/master/jigsaw#L28) folders.

**Usage Example**
I would like to build a static site out of pieces of my dynamic site. Therefore I would love for it to share the same layouts, partials, etc., but don't want to put all of my view files in your `/source` folder. :wink: 

I'm more than willing to code this up...just wanted to clear it with you first before I proceed...

**Comment by adamwathan** [comment]
How are you thinking it would work, optional command line flag? Definitely
think it's a reasonable idea. Is your use case solvable with a symlink or
is it more complex than that?

**Comment by underparnv** [comment]
Yeah...an optional argument I think would work best. I fiddled with the idea a bit this afternoon and found that it will be somewhat difficult with the `jigsaw` file in that we would have to bind the paths after the command is invoked. There is a `getFinder` method on the Illuminate ViewFactory which I'm pretty sure is the way to go...just haven't had time to complete that connection. Thanks for the reply!
